src/modules/dashboard/components/ColonyHome/ColonySubscription/ColonySubscription.css

```diff
@@ -33,10 +33,12 @@
 }
 
 .menuIconContainer {
```

Comment:
Also this now doesn't need all the styles shared with the main component, eg `position: relative`, `background`, `cursor` etc.